docs/paper/dev/getting-started/userdev.md

```diff
@@ -0,0 +1,96 @@
+---
+slug: /dev/userdev
+---
+
+# paperweight-userdev
+
+**paperweight** is the name of Paper's custom build tooling. The **paperweight-userdev** Gradle plugin part of that
+provides access to internal code (also known as NMS) during development.
+
+:::note
+
+This guide is written using the Kotlin DSL for Gradle and assumes you have some basic knowledge of Gradle.
+If you want to see a fully-functioning plugin that uses **paperweight-userdev**,
+check out this [example plugin](https://github.com/PaperMC/paperweight-test-plugin).
+
+:::
+
+## Why this is useful
+The Paper server jars we provide on the downloads page through the API are **paperclip** jars. These
+use Spigot's mappings, which are essentially some type names, but fully obfuscated fields and methods.
+This can make it hard to work with in a development environment. This plugin lets you use fully deobfuscated
+types, names and fields during development, and then remaps your plugin, so it can still be used with the obfuscated
+server.
+
+:::caution
+
+The re-obfuscation does not apply to reflection. Look at something like [this library](https://github.com/jpenilla/reflection-remapper) to be able to
+use non-obfuscated names in reflection.
+
+:::
+
+## Adding the plugin
+Add the plugin to your `build.gradle.kts` file.
+```kotlin
+plugins {
+    id("io.papermc.paperweight.userdev") version "1.5.5" // the latest version can be found on the Gradle Plugin Portal
+}
+```
+
+:::info[Snapshots]
+
+**paperweight-userdev** releases are available through the Gradle plugin portal, but if you
+want to use SNAPSHOT versions, you must add Paper's maven repo to `settings.gradle.kts` with:
+```kotlin
+pluginManagement {
+    repositores {
+        maven("https://repo.papermc.io/repository/maven-public/")
+    }
+}
+```
+
+:::
+
+## Adding the dev-bundle dependency
+If you try to load your Gradle project now, you will receive an error saying you have to declare
+a dev bundle dependency. You can do that by adding to your `dependencies` block in your `build.gradle.kts`
+file.
+
+```kotlin
+dependencies {
+    // other deps
```

Comment:
I think I would rather this said either `// Other Dependencies` or just `// ...`